.github/workflows/dev_on_workflow_tester.yml

```diff
@@ -86,18 +87,32 @@ jobs:
           key: v1-yarn-dependency-cache-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             v1-yarn-dependency-cache-${{ hashFiles('**/yarn.lock') }}
-      - run:  yarn install --immutable --cwd server
+      - run: yarn install --immutable --cwd server
       - uses: actions/cache@v3
         with:
           path: .jest/cache
           key: v1-yarn-test-native-cache-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             v1-yarn-test-native-cache-${{ hashFiles('**/yarn.lock') }}
-      - run: yarn --cwd server test:unit:ci 
+      - run: yarn --cwd server test:unit:ci
       - name: SonarCloud scan
         uses: SonarSource/sonarcloud-github-action@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         with:
           projectBaseDir: server
+
+  yarn_test_storybook:
+    runs-on: ubuntu-latest
```

Comment:
self-hosted plutot, cf la déclaration des images plus haut

Reply:
Ça ne fonctionne pas, comme pour le build and deploy de storybook. On avait dit qu'on restait sur ubuntu-latest ?